src/public/js/rs.js

```diff
@@ -2,6 +2,8 @@
 var RS = (function() {
   // (actually ROT-13 right now, as a dummy example. We can replace w/ RS)
   var ROT = 13;
+  var GEN = 3;
+  var GF = 256;
 
   function encode(msg) {
```

Comment:
you  moved the shift function to util -- wanna get rid of it in rs.js and switch encode / decode to using that version? (doesn't really matter since we'll get rid of the rot13 stuff eventually, but duplicate code)